Sources/Exporters/DataCompression/DataCompression.swift

```diff
@@ -28,9 +28,11 @@
 
 
 import Foundation
+
+#if canImport(Compression)
 import Compression
```

Comment:
Looks like macOS and Linus both fail with that solution. @nachoBonafonte @bryce-b let me know if there is anywhere else I need to address the platform issue. Most probably in the `Package.swift` file but not entirely sure where

Reply:
I think the issue is you're importing DataCompression in the exporters when there's nothing there due to the #if

Reply:
Looks Test was a blocker. Fixed it. Thanks for the tip!